alarms_agent.cpp

```diff
@@ -109,16 +150,14 @@ int main (int argc, char **argv)
   init_ituAlarmTable();
 
   // Run forever
-  init_snmp("clearwater-alarms");
+  init_snmp_handler_threads("clearwater-alarms");
```

Comment:
Ah, this is the new SNMP setup code you alluded to. I was confused because it's been put in the global namespace and is inconsistently named (`snmp_setup` sounds like a built in netsnmp function to me). I'll stop whinging now.